images/multus-daemonset.yml

```diff
@@ -117,6 +117,19 @@ spec:
       - operator: Exists
         effect: NoSchedule
       serviceAccountName: multus
+      initContainers:
+      - name: install-cni-binaries
+        image: rancher/coreos-flannel-cni:v0.3.0
```

Comment:
Change to `nfvpe/containernetworking-plugins:latest`

Reply:
I'd prefer to use a version-tagged plug-in, see my comments below.